binderhub/app.py

```diff
@@ -107,6 +107,17 @@ def _log_level(self):
         config=True
     )
 
+    banner_message = Unicode(
+        '',
+        help="""
+        Message to display in a banner on all pages.
+
+        Will be directly inserted into the about page's source so you can use
+        raw HTML.
```

Comment:
I suggest the following rephrasing as I got surprised about "directly inserted" in practice meant a container with grey background would wrap the content.

"The content of `banner_message` will be inserted as it is within a `<div>` with grey background, located at the top of the BinderHub pages. Raw HTML is supported."

Reply:
Good point, also noticed the docstring was still referring to "about message" so fixed that as well.